scripts/buildtools/webpack/plugins.js

```diff
@@ -65,6 +68,21 @@ function plugins(options = {}) {
       emptyModule(),
       devMode(),
     ];
+  
+  if (copyAssets) {
+    list.push(new EventHooksPlugin({
```

Comment:
Could this accomplished via https://webpack.js.org/plugins/copy-webpack-plugin/?

I'd like to reduce the amount of 3p plugins we introduce